integration/js/pages/AppPage.js

```diff
@@ -65,12 +65,11 @@ function findAllElements() {
 
     eventReportingCheckBox: By.id('event-reporting'),
     eventReportingCheckBoxLabel: By.css('label[for="event-reporting"]'),
-    backgroundBlurFilterButton: By.id('dropdown-menu-filter-Background-Blur-10%-CPU'),
+    backgroundBlurFilterButton: By.id('dropdown-menu-filter-Background-Blur-40%-CPU'),
+    backgroundReplacementFilterButton: By.id('dropdown-menu-filter-Background-Replacement'),
     microphoneDropEchoButton: By.id('dropdown-menu-microphone-Echo'),
     echoReductionFeature: By.id('echo-reduction-capability'),
-    echoReductionFeatureLabel: By.css('label[for="echo-reduction-capability"]'),
-    stereoMusicProfileCheckBox: By.id('fullband-music-stereo-quality'),
```

Comment:
Why do we want to remove these musicProfile checkbox?

Reply:
bad merge. Updated